app/controllers/donuts_controller.rb

```diff
@@ -21,6 +21,10 @@ def create
 
   def show
     @donut = Donut.find(params[:id])
+    @reviews = @donut.reviews
+    @url = "donut_reviews_path"
+    @method = "post"
```

Comment:
Reviews are nested inside the Donut#show page. Like yelp, someone can add a review at the bottom of the page. Eventually we will have links in line with a users review that allows them to edit the review on the same page